configs/laramie_bmi_config_coupler.txt

```diff
@@ -0,0 +1,9 @@
+soil_params.smcmax=0.439
```

Comment:
Do we need to add units to this config file similar to the cfe config? [units]

Reply:
yep, I have added units to the input parameters.